mandatory/3-playing-computer.js

```diff
@@ -7,12 +7,19 @@
   Answer the following questions:
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
+  Answer: Line 32 throws an error because "b" is not defined
   2. Remove the line that throws the error.
+  Done
   3. What is printed to the console?
+  Answer: The values a and x and the results of f1 and f2 functions.
   4. How many times is "f1" called?
+  Answer: 2 times
   5. How many times is "f2" called?
+  Answer: 3 times
   6. What value does the "a" parameter take in the first "f1" call?
+  Answer: 7
```

Comment:
this exercise is very unclear because it reuses the same variable names. If you had a look at the solutions you will see this answer is `1` because the `a` the question is referring to is the parameter in the function definition for `f1`, i.e. this `a`
```
const f1 = function (a, b) {
```

Reply:
I have now understood why it is 1 but it is confusing :)